tests/test_modeling_tf_common.py

```diff
@@ -236,6 +243,43 @@ def test_onnx_compliancy(self):
                     incompatible_ops.append(op)
 
             self.assertEqual(len(incompatible_ops), 0, incompatible_ops)
+    
+    @require_onnx
+    @slow
+    def test_onnx_runtime_quantize_optimize(self):
+        if not self.test_onnx:
+            return
+
+        import onnx
+        import onnxruntime
+        from onnxruntime.quantization import QuantizationMode, quantize
+        import keras2onnx
+
+        config, inputs_dict = self.model_tester.prepare_config_and_inputs_for_common()
+
+        for model_class in self.all_model_classes:
+            model = model_class(config)
+            model(model.dummy_inputs)
+
+            onnx_model = keras2onnx.convert_keras(model, model.name, target_opset=self.onnx_min_opset)
+
+            with tempfile.TemporaryDirectory() as tmpdirname:
+                keras2onnx.save_model(onnx_model, os.path.join(tmpdirname, "model.onnx"))
+                sess_option = onnxruntime.SessionOptions()
+                sess_option.optimized_model_filepath = os.path.join(tmpdirname, "model-optimized.onnx")
+                onnxruntime.InferenceSession(os.path.join(tmpdirname, "model.onnx"), sess_option)
```

Comment:
Ok, it's the old way of quantizing the model, and not the one recommended anymore which might lead to strange result.

I would not try quantization here as it's hardware dependant and we have no control on the machine actually running this test.